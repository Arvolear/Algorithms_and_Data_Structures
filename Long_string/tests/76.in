239402504483322663029471483727860800272811
4552292409115142533133