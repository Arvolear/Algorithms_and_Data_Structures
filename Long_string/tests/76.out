239402504483322663034023776136975942805944
239402504483322663024919191318745657739678
1089830203882583636811322139430108457361196526942323036006546863
52589439115106584488